### DISCUSSION TITLE
Get the err msg while fetching changelogs - "Unknown platform, skipping changelog fetching."

### DISCUSSION THREAD

**Discussion by nchery** [renovatebot/renovate#20248] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.121.0

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

5.9.0

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'd like the release notes from the changelogs file of the monorepo root folder posted into the PRs created

### What you tried so far.

Debug logs ---> see logs below

I found out this thread [discussion 15265](https://github.com/renovatebot/renovate/discussions/15265) where it is mentioned that release notes works only on Github and Gitlab. is that correct ? 

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Fetching changelog: https://bitbucket.babel.fr/repoURL.git (4.1.2 -> 4.2.3) (repository=REPO, baseBranch=dev, branch=renovate/@NAME)
 INFO: Unknown platform, skipping changelog fetching. (repository=REPO, baseBranch=dev, branch=renovate/@NAME)
       "sourceUrl": "https://bitbucket.babel.fr/repoURL.git",
       "hostType": null
```

</details>

**Comment by viceice** [comment, marked as answer]
yes, currently only GitHub and Gitlab
